--- raw-cpuid-11.0.2/Cargo.toml	1970-01-01T00:00:01+00:00
+++ raw-cpuid-11.0.2/Cargo.toml	2024-05-28T17:37:01.449943+00:00
@@ -32,11 +32,6 @@
 license = "MIT"
 repository = "https://github.com/gz/rust-cpuid"
 
-[[bin]]
-name = "cpuid"
-path = "src/bin/cpuid.rs"
-required-features = ["cli"]
-
 [[example]]
 name = "serialize_deserialize"
 path = "examples/serialize_deserialize.rs"
@@ -44,11 +39,6 @@
 
 [dependencies.bitflags]
 version = "2.0"
-
-[dependencies.clap]
-version = "4.2"
-features = ["derive"]
-optional = true
 
 [dependencies.serde]
 version = "1.0"
@@ -63,21 +53,7 @@
 version = "1.0"
 optional = true
 
-[dependencies.termimad]
-version = "0.25"
-optional = true
-
 [features]
-cli = [
-    "display",
-    "clap",
-]
-display = [
-    "std",
-    "termimad",
-    "serde_json",
-    "serialize",
-]
 serialize = [
     "serde",
     "serde_derive",
