--- ndarray-0.15.6/Cargo.toml	1970-01-01T00:00:01+00:00
+++ ndarray-0.15.6/Cargo.toml	2024-05-28T16:19:19.714821+00:00
@@ -54,7 +54,7 @@
 bench = false

 [dependencies.approx]
-version = "0.4"
+version = "0.5"
 optional = true
 default-features = false

@@ -105,7 +105,7 @@
 default-features = false

 [dev-dependencies.approx]
-version = "0.4"
+version = "0.5"

 [dev-dependencies.defmac]
 version = "0.2"
