--- metrics-util-0.17.0/Cargo.toml	1970-01-01T00:00:01+00:00
+++ metrics-util-0.17.0/Cargo.toml	2024-08-27T08:10:55.824880+00:00
@@ -119,7 +119,7 @@
 default-features = false
 
 [dependencies.sketches-ddsketch]
-version = "0.2"
+version = "0.3"
 optional = true
 default-features = false
 
@@ -144,10 +144,10 @@
 version = "0.11"
 
 [dev-dependencies.ndarray]
-version = "0.15"
+version = "0.16"
 
 [dev-dependencies.ndarray-stats]
-version = "0.5"
+version = "0.6"
 
 [dev-dependencies.noisy_float]
 version = "0.2"
@@ -156,10 +156,10 @@
 version = "4.2"
 
 [dev-dependencies.predicates-core]
-version = "=1.0.5"
+version = "1.0.5"
 
 [dev-dependencies.predicates-tree]
-version = "=1.0.7"
+version = "1.0.7"
 
 [dev-dependencies.quickcheck]
 version = "1"
@@ -175,7 +175,7 @@
 version = "0.4"
 
 [dev-dependencies.sketches-ddsketch]
-version = "0.2"
+version = "0.3"
 
 [dev-dependencies.tracing]
 version = "0.1"
